ddtrace/llmobs/_constants.py

```diff
@@ -103,4 +103,6 @@
 
 EXPERIMENT_ID_KEY = "_ml_obs.experiment_id"
 EXPERIMENT_EXPECTED_OUTPUT = "_ml_obs.meta.input.expected_output"
+EXPERIMENTS_INPUT = "_ml_obs.meta.input"
+EXPERIMENTS_OUTPUT = "_ml_obs.meta.output"
```

Comment:
If we're going to have this key, can we just name it `FREEFORM_INPUT` etc? Or is this not available in regular LLMObs spans except for experiment spans

Reply:
only available in experiments spans for now